### ISSUE TITLE
Data/sim and wifi icon swap not working

### ISSUE THREAD

**Issue by roydhritiman** [MonwF/customiuizer#383]
Hello

I tried using CustoMIUIzer14 to swap the data and wifi icons on the right as I dislike the default icon layout of MIUI.

The swap works, but ONLY if i shift all icons to the left. But it doesn't work when all the icons are on the right (default).

I want the swap mod to work on the right side of the status bar. Can you fix this?

Attaching two screenshots:

![ade0751e-deae-4c15-90ec-cfa18bd85491](https://github.com/MonwF/customiuizer/assets/38285763/405ebc89-7609-43a3-bf03-2222245a6fd2)
![0f89e96b-a1af-4262-a49e-e46ed7ce4088](https://github.com/MonwF/customiuizer/assets/38285763/5aa3d2f4-c11f-4beb-a826-22e86633f02b)

**Comment by MonwF** [comment]
yeah, someone reported this bug. but it works on my device. can you provide system info and logs、systemui.apk?

**Comment by roydhritiman** [comment]
![Screenshot_20231027-104554_cropped](https://github.com/MonwF/customiuizer/assets/38285763/78afd0b8-cd87-4ac2-8ef7-20c64ab8a070)
Wow, its somehow working after I flashed the modded version of the universal safetynet fix. I genuinely don't get how lmao.

**Comment by MonwF** [comment]
glad to see that. i released a test build for new systemui
